lte/gateway/python/magma/pipelined/app/policy_mixin.py

```diff
@@ -334,6 +351,14 @@ def _get_classify_rule_flow_msgs(self, imsi, ip_addr, apn_ambr, flow, rule_num,
                 copy_table=copy_table,
                 resubmit_table=next_table)
             )
+
+        if self.proxy_controller:
```

Comment:
maybe

if self.proxy_controller and urls:

Reply:
The url check is done in he controller.